### DISCUSSION TITLE
OLEd

### DISCUSSION THREAD

**Discussion by Thai7Ken** [Yowkees/keyball#708] [Q&A]
半田付け中にOLEDを焼いてしまい、表示できなくなってしまいました。
新しくディスプレイとピン等を購入できますでしょうか？
また、OLED無しでどんな支障があるかも教えていただければ幸いです。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

送付先等やりとりさせていただきたいので、下記白銀ラボウェブショップのお問い合わせよりご連絡いただければ幸いです。
https://shirogane-lab.net/inquiry

以上、よろしくお願いいたします。
